src/makeRobot.js

```diff
@@ -38,7 +38,66 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return (`name: ${this.name},`
+      + ` chip version: ${this.version},` + ` wheels: ${this.wheels}`);
```

Comment:
You can do like this

```suggestion
      return (
        `name: ${this.name},  chip version: ${this.version},  wheels: ${this.wheels}`
      );
```